admin/src/components/list/index.js

```diff
@@ -109,19 +109,26 @@ export const ResultTable = styled.div`
     }
   }
   select.sort-options {
-    background: none; // remove the arrows added by sortOptions
+    background: transparent;
+    background-image: url("data:image/svg+xml;utf8,<svg fill='black' height='24' viewBox='0 0 24 24' width='24' xmlns='http://www.w3.org/2000/svg'><path d='M7 10l5 5 5-5z'/><path d='M0 0h24v24H0z' fill='none'/></svg>");
+    background-repeat: no-repeat;
+    background-position-x: 100%;
+    background-position-y: 5px;
```

Comment:
3 seems better

```suggestion
    background-position-y: 3px;
```
<img width="475" alt="Capture d’écran 2021-10-07 à 15 49 54" src="https://user-images.githubusercontent.com/1575946/136397655-58d3b7a2-266e-4e55-960b-0d32d439f49a.png">